forms/base_form.go

```diff
@@ -0,0 +1,10 @@
+package forms
+
+var ValidationMessages = map[string]string{
+	"ConfirmPassword":   "Confirm password confirmation must match",
+	"ExistingEmail":     "Email is already in use",
+	"InvalidCredential": "Your email or password is incorrect",
+	"InvalidLinkList":   "All Link list must be valid URL",
+	"InvalidVisitUrl":   "Visit URL must be valid URL",
+	"NotExistingUser":   "User does not exist",
```

Comment:
How about making the naming slightly more generic:

```suggestion
	"InvalidUser":   "User does not exist",
```

This naming would also be more consistent with the other constants.